.gitignore

```diff
@@ -13,3 +13,4 @@
 Makefile*
 mkmf.log
 /Gemfile.lock
+/.idea
```

Comment:
First time a contributor uses RubyMine I see

Reply:
@ObiWanKeoni no, it is just that you didn't configure your global gitignore 😉 

There are three levels of ignoring in git:

1. `.gitignore`, corresponding to what the project want to ignore
2. `.git/info/exclude`, local to a projet and a specific person
3. `~/.gitignore` (or other location configurable) that is computer wide.

I think `/.idea` belongs to the third, and not this project!

Reply:
I learn something new about git every week it seems like lol

thanks for the context, didn't realize this was a thing!